server/internal/apikey/cache.go

```diff
@@ -81,9 +83,18 @@ func (c *Cache) updateCache(ctx context.Context) error {
 
 	m := map[string]*K{}
 	for _, apiKey := range resp.Data {
+		var usrID, orgID string
+		if apiKey.User != nil {
```

Comment:
Question: Do we need this `nil` check? Just wondering if we can always assume that `User` is not `nil`. We don't need to worry backward compatibility now.

Reply:
No, it's just for backward compatibility. I'll remove both checks.